content/en/docs/tasks/observability/metrics/tcp-metrics/index.md

```diff
@@ -91,15 +91,17 @@ example configuration and commands.
 
 1.  Send traffic to the sample application.
 
-    Set `$GATEWAY_URL` using [these instructions](/docs/setup/getting-started/#determining-the-ingress-ip-and-ports)
-
     For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
     browser or use the following command:
 
     {{< text bash >}}
     $ curl http://$GATEWAY_URL/productpage
     {{< /text >}}
 
+    {{< tip >}}
+    `$GATEWAY_URL` is the value set in the [Bookinfo](/docs/examples/bookinfo/) example.
```

Comment:
I think pointing users to the right section would be more helpful.

So this 
`/docs/examples/bookinfo/`

Should be this
`docs/setup/getting-started/#determining-the-ingress-ip-and-ports`

Reply:
We had this before but it was confusing to folks.. see removed changes please... 
Ref: https://github.com/istio/istio.io/issues/7184